symmetric-db/src/main/java/org/jumpmind/db/platform/mssql/MsSql2000DdlBuilder.java

```diff
@@ -169,6 +170,10 @@ protected void dropTable(Table table, StringBuilder ddl, boolean temporary, bool
         println("  CLOSE refcursor", ddl);
         println("  DEALLOCATE refcursor", ddl);
         ddl.append("  DROP TABLE ");
+        if (schema != null) {
```

Comment:
Can you change this to if StringUtils.isNotBlank(schema) instead?

Reply:
I've changed it as you suggested